### DISCUSSION TITLE
I miss an imported icon ... 😄

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#77] [Q&A] [answered]
Just tried to meet some people at the favorite parking lot:
![2023-03-10 16 25 47](https://user-images.githubusercontent.com/121806238/224357418-23bf9250-074a-4fd7-a5a3-cda6d40d24e5.png)   ![2023-03-10 16 26 33](https://user-images.githubusercontent.com/121806238/224357959-4dde4812-4d27-453c-bc55-c3613cfc066d.png)   ![2023-03-10 16 26 42](https://user-images.githubusercontent.com/121806238/224357995-2f6ad5db-05bd-4c35-865d-4365dfb84895.png)
Cruiser 3.0.12_____________________I miss this (Kurviger 2)____________to implement it into the route

Or have I to switch something on?

**Comment by devemux86** [comment, marked as answer]
This button centered the map on the selected favorite / waypoint and opened its long press dialog.
As this functionality is already available, the button was removed to make the toolbar less cluttered.

If you select a favorite / waypoint, then press the OK button, the map will be centered on that point.
Do we need its long press dialog to open automatically or leave the action to the user?

**Comment by devemux86** [comment]
Nice fuel symbol in the names, is it a unicode character? :fuelpump:

**Comment by devemux86** [comment]
In the future it will be easier to find favorites and locate them on the map with the new Search: https://github.com/devemux86/cruiser/discussions/74.

**Comment by WolfgangZ42** [comment]
Sometimes it's quite logical ... 
Sorry, it should have been intuitive for me too!

> Nice` fuel symbol in the names, is it a unicode character? ⛽

It's just an emoji - but also an unicode character: U+26FD
